sycl/include/sycl/ext/intel/experimental/kernel_properties.hpp

```diff
@@ -10,72 +10,58 @@
 
 #pragma once
 
-#include <sycl/ext/intel/experimental/detail/misc_intrin.hpp>
-
-#include <sycl/detail/boost/mp11.hpp>
+#include <sycl/ext/oneapi/properties/property.hpp>
+#include <sycl/ext/oneapi/properties/property_value.hpp>
 
 namespace sycl {
 __SYCL_INLINE_VER_NAMESPACE(_V1) {
-namespace ext::intel::experimental {
-
-namespace kernel_properties {
-
-/// <summary>
-///  This namespace contains APIs to set kernel properties.
-/// </summary>
+namespace ext {
+namespace intel {
+namespace experimental {
```

Comment:
Nit; We can just define them in one line now, i.e.

```suggestion
namespace ext::intel::experimental {
```

It's not a requirement, but I like it.

Reply:
thanks, fixed in latest commit